solr/src/main/resources/solr/conf/search-solr-managed-schema.xml

```diff
@@ -116,19 +116,20 @@
     <!-- docValues are enabled by default for long type so we don't need to index the version field  -->
     <field name="_version_" type="plong" indexed="false" stored="false"/>
 
-    <!-- CST fields -->
-    <field name="contents" type="text_general" indexed="true" stored="true" required="true"
-           multiValued="false" />
-    <field name="irm_uri_string" type="string" indexed="true" stored="true" required="true"
-           multiValued="false" />
-
-    <!-- If you don't use child/nested documents, then you should remove the next two fields:  -->
-    <!-- for nested documents (minimal; points to root document) -->
-    <field name="_root_" type="string" indexed="true" stored="false" docValues="false" />
-    <!-- for nested documents (relationship tracking) -->
-    <field name="_nest_path_" type="_nest_path_" /><fieldType name="_nest_path_" class="solr.NestPathField" />
-
-    <field name="_text_" type="text_general" indexed="true" stored="false" multiValued="true"/>
+  <!-- CST fields -->
```

Comment:
Added some fields. Use default field properties to simplify the declarations.
See https://lucene.apache.org/solr/guide/8_1/field-type-definitions-and-properties.html#field-default-properties